src/coreclr/jit/codegenloongarch64.cpp

```diff
@@ -0,0 +1,8863 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Copyright (c) Loongson Technology. All rights reserved.
```

Comment:
Copyright message like this is not standard for new files. Could you please delete it if possible? The contribution license terms are covered by [CLA](https://github.com/dotnet/runtime/blob/main/CONTRIBUTING.md#contributor-license-agreement).

Reply:
> Copyright message like this is not standard for new files. Could you please delete it if possible? The contribution license terms are covered by [CLA](https://github.com/dotnet/runtime/blob/main/CONTRIBUTING.md#contributor-license-agreement).

OK.